codeland.html

```diff
@@ -1,12 +1,101 @@
+        </defs>
+        </svg>
+      <span>The community-first conference designed to level you up.</span>
+    </div>
+  </div>
+
+  <div class="codeland-row h-flex">
+    <iframe id="livestream" src="https://www.youtube.com/embed/EEIk7gwjgIM?controls=0&rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture;" allowfullscreen></iframe>
+
+    <div id="speaker-info" class="v-flex">
+      <p class="title ff-accent fs-l fw-bold">About the Speaker</p>
+      <p class="body">
+        Vaidehi Joshi is a senior engineer at DEV, where she builds community and helps improve the software careers of millions. She enjoys building and breaking code, but loves creating empathetic engineering teams a whole lot more. She is the creator of basecs and baseds, two writing series exploring the fundamentals of computer science and distributed systems. She also co-hosts the Base.cs Podcast, and is a producer of the BaseCS and Byte Sized video series.
+      </p>
+      <button class="crayons-btn crayons-btn--secondary" type="button">See discussion</button>
+    </div>
```

Comment:
Crayons being used here! Since the page will be embedded in the shell we have access to it.

The more we rely on using these existing components the better I would think 👀

Reply:
@fdoxyz Phew, it's good to hear that we can use Crayons. I was worried for a sec...